web-client/vite.config.ts

```diff
@@ -12,6 +13,9 @@ import { normalizePath } from "vite";
 export default defineConfig({
   server: {
     strictPort: true,
+    headers: {
+      "Content-Security-Policy": generateCSP(true),
```

Comment:
Is it meant to be always true?

Reply:
yeah, this looks like it should probably be something like process.env.NODE_ENV
```ts
    headers: {
      "Content-Security-Policy": generateCSP(process.env.NODE_ENV !== "production"),
```

Reply:
We only use the Vite server in dev mode.
In prod the build is server statically. The CSP in production comes from `netlify.toml` That's why here we have CSP enabled (so things get blocked from day 1 in dev) but in `netlify.toml` it's `CSP-Report-Only`.

You guys raised a good point of future compatibility and readbility though. I'll switch